tools/Linux/kodi.sh.in

```diff
@@ -44,12 +48,48 @@ do
             LIBDIR="$2"
             shift; shift
             ;;
+        --windowing)
+            WINDOWING="$2"
+            echo "$SAVED_ARGS" | sed "s/--windowing[ ]*$2//g"
+            shift; shift
+            ;;
         *)
             shift
             ;;
     esac
 done
 
+# Note: by default only one of those binaries exists
+# To be able to select a different one, if has to be compiled specifically
+# Your distribution might provide extra packages for those
+case "$WINDOWING" in
```

Comment:
Why not just `KODI_BINARY=$LIBDIR/${bin_name}/${bin_name}-${WINDOWING}` and handle the special case of `auto` beforehand by setting `WINDOWING` to `wayland`? That way we don't have to touch the script when changing available platforms.